modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/js/main.es.js

```diff
@@ -312,6 +327,53 @@ class Form extends Component {
 
 		this._eventHandler.removeAllListeners();
 
+		const addFieldButton = document.getElementById('addFieldButton');
+
+		if (addFieldButton) {
+			addFieldButton.removeEventListener(
+				'click',
+				this._handleAddFieldButtonClicked.bind(this)
```

Comment:
Similar issue here: `fn.bind(this) !== fn.bind(this)`, so this won't work.

You should probably check every change you've made in this file (and in this PR) for tricky `bind`. Basically, you need to save the result of `bind` in a property so that you can retrieve it later and dispose of the listener properly.